Detections/VectraAI/VectraDetect-Suspected-Behavior-by-Tactics.yaml

```diff
@@ -0,0 +1,79 @@
+  - connectorId: AIVectraDetect
+    dataTypes:
+      - CommonSecurityLog
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+  - Discovery
+  - LateralMovement
+  - Collection
+  - CommandAndControl
+  - Exfiltration
+  - Impact
+relevantTechniques:
+query: |
+  // Edit this variable to only keep the tactics where an incident needs to be created (Defaults are: "COMMAND & CONTROL", "BOTNET ACTIVITY", "EXFILTRATION", "LATERAL MOVEMENT", "RECONNAISSANCE")   
+  let configured_tactics = dynamic(["COMMAND & CONTROL", "BOTNET ACTIVITY", "EXFILTRATION", "LATERAL MOVEMENT", "RECONNAISSANCE"]);
+  CommonSecurityLog
+  | where DeviceVendor == "Vectra Networks"
+  | where DeviceProduct == "X Series"
+  | where DeviceEventClassID != "campaigns" and DeviceEventClassID != "hsc" and DeviceEventClassID != "audit" and DeviceEventClassID != "health" and DeviceEventClassID != "asc" 
+  | extend Category = extract("cat=(.+?);", 1, AdditionalExtensions) 
+  | project-rename threat_score = FlexNumber1
+  | project-rename certainty_score = FlexNumber2
+  | project-rename triaged = DeviceCustomString5
+  | project-rename vectra_URL = DeviceCustomString4
+  | project-rename detection_name = DeviceEventClassID
+  | extend Tactic = case( Category == "COMMAND & CONTROL", "CommandAndControl",
+                          Category == "BOTNET ACTIVITY" , "Impact",
+                          Category == "EXFILTRATION", "Exfiltration",
+                          Category == "LATERAL MOVEMENT", "LateralMovement",
+                          Category == "RECONNAISSANCE", "Discovery",
+                          "UNKNOWN")
+  | extend level = case( threat_score <  50 and certainty_score < 50, "Low",
+                         threat_score < 50 and certainty_score >= 50 , "Medium", 
+                         threat_score >= 50 and certainty_score <= 50, "High", 
+                         threat_score >= 50 and certainty_score >= 50, "Critical",
+                         "UNKNOWN")
+  | extend Severity = case( level == "Low", "Low",
+                            level == "Medium", "Medium",
+                            level == "High", "Medium",
+                            level == "Critical", "High",
+                            "UNKNOWN")
+  | extend account = extract("account=(.+?);", 1, AdditionalExtensions)
+  | extend account_consolidated = iff(account matches regex ":", tostring(split(account,":")[1]) ,tostring(split(account,":")[0])) 
+  | extend source_entity = case( isnotempty(account_consolidated), account_consolidated,
+                                 isnotempty(SourceHostName), SourceHostName,
+                                 "UNKNWON") 
+  | where Category in (configured_tactics) 
+  | summarize arg_max(threat_score, *) by source_entity , Activity
+  | project  source_entity, SourceHostName, SourceIP, account_consolidated, Activity, Tactic, Severity, threat_score, certainty_score, triaged, vectra_URL
```

Comment:
Do we want to map the different entity in the end using extend.
https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#old-entity-mapping-method

Also wondering, if these logs have a Timestamp that we need to bring in the results as well as extend in the final line of the query.